content/faq.ja.md

```diff
@@ -1,8 +1,8 @@
 ---
 html: faq.html
 parent: xrp-ledger-overview.html
-blurb: バリデータ、ユニークノードリスト、XRPの役割、セキュリティなどのトピックに関するよくある質問に対しての答えを見つけてください。
-subtitle: Your Questions About XRPL, Answered
+blurb: XRPレジャー、XRPLエコシステム、コミュニティに関するよくある質問にお答えします。
```

Comment:
I would like to keep "XRP Ledger" as a whole name instead of putting "Ledger" in katakana (here and elsewhere in the doc).

```suggestion
blurb: XRP Ledger、XRPLエコシステム、コミュニティに関するよくある質問にお答えします。
```